tests/unit/components/buttons/HdRadioButton.spec.js

```diff
@@ -11,7 +11,9 @@ describe('HdRadioButton', () => {
   const name = 'lorem';
   const value = 'ipsum';
   const label = 'dolor';
-  const mockedSelect = jest.fn();
+  const ICON_SELECTOR = '.radioButton__icon';
+  const BACKGROUND_STYLE = 'background-image: url(desktop-icon-url);';
```

Comment:
```suggestion
  const BACKGROUND_STYLE = `background-image: url(${desktopIcon});`;
```

Reply:
I strongly recommend updating to @ilyasmez instead of using hard-coded strings. We define this`desktop-icon-url` above.

Reply:
I have removed this variable, since I'm using "stringContaining" as suggested below.